docs/METRICS_AND_SOURCES.md

```diff
@@ -50,7 +50,8 @@ This document lists all [metrics](#metrics) that *Quality-time* can measure and
 | [Cobertura Jenkins plugin](https://plugins.jenkins.io/cobertura/) | Jenkins plugin for Cobertura, a free Java tool that calculates the percentage of code accessed by tests. | [Source up-to-dateness](#source-up-to-dateness-from-cobertura-jenkins-plugin), [Test branch coverage](#test-branch-coverage-from-cobertura-jenkins-plugin), [Test line coverage](#test-line-coverage-from-cobertura-jenkins-plugin) |
 | [Composer](https://getcomposer.org/) | A Dependency Manager for PHP. | [Dependencies](#dependencies-from-composer) |
 | [GitLab](https://gitlab.com/) | GitLab provides Git-repositories, wiki's, issue-tracking and continuous integration/continuous deployment pipelines. | [Failed CI-jobs](#failed-ci-jobs-from-gitlab), [Source up-to-dateness](#source-up-to-dateness-from-gitlab), [Unmerged branches](#unmerged-branches-from-gitlab), [Unused CI-jobs](#unused-ci-jobs-from-gitlab) |
-| [JUnit XML report](https://junit.org) | Test reports in the JUnit XML format. | [Source up-to-dateness](#source-up-to-dateness-from-junit-xml-report), [Tests](#tests-from-junit-xml-report) |
+| [Generic Security Warnings](https://github.com/ICTU/quality-time/blob/master/docs/GENERIC_REPORTS.MD) | This source collector is used for Generic Security Warnings imported from json files | [Security warnings](#generic-security-warnings) |
+| [JUnit XML report](https://junit.org) | Test reports in the JUnit XML format. | [Security warnings](#security-warnings-from-snyk) |
```

Comment:
This probably needs to be updated by running `docs/ci/create_metrics_and_sources_md.py` again.